pypesto/history/amici.py

```diff
@@ -0,0 +1,232 @@
+from pathlib import Path
+from typing import Sequence, Union
+
+import numpy as np
+
+from ..C import (
+    RDATAS,
+    CPU_TIME_TOTAL,
+    PREEQ_CPU_TIME,
+    POSTEQ_CPU_TIME_B,
+    POSTEQ_CPU_TIME,
+    PREEQ_CPU_TIME_B
+)
+from .csv import CsvHistory
+from .hdf5 import Hdf5History
+from .options import HistoryOptions
+from .util import trace_wrap
+
+
+class Hdf5AmiciHistory(Hdf5History):
+    """
+    Stores a representation of the history in an HDF5 file, extended with
+    AMICI-specific traces of total simulation time, pre-equilibration time and
+    post-equilibration time.
+
+    Parameters
+    ----------
+    id:
+        Id of the history
+    file:
+        HDF5 file name.
+    options:
+        History options. Defaults to ``None``.
+    """
+
+    def __init__(
+        self,
+        id: str,
+        file: Union[str, Path],
+        options: Union[HistoryOptions, dict, None] = None,
+    ):
+        super().__init__(id, file, options=options)
+
+    @staticmethod
+    def _simulation_to_values(x, result, used_time):
+        values = Hdf5History._simulation_to_values(x, result, used_time)
```

Comment:
Here it's `@staticmethod`, but n `CsvAmiciHistory` it's `super()...`. Just curious, why?

Reply:
In `CsvHistory` it's not static as it uses dynamically updated attributes (e.g. `_n_fval`), in `Hdf5History` it's not the case, so I made the methods static both in `Hdf5History` and `Hdf5AmiciHistory`.